tests/yam/validate/validate_suseconnect_url.pm

```diff
@@ -13,7 +13,8 @@ use utils;
 
 sub run {
     select_console 'root-console';
-    my $scc_url = get_required_var('SCC_URL');
+    my $agama_profile_options_regurl = qr/registration_url="(?<registration_url>.+)"/;
```

Comment:
let's not declare variable in general and use the expression below directly, it is better practice to avoid that, speciall when there is not more than one use.
you could have an enter after `?` and separate each branch of the condition in different lines for example to shorten the line.